### PR TITLE
DOCS-7485-update applying policies prereqs

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

**Review by nahuel-dallavecchia — APPROVED** [review]

File: modules/ROOT/pages/automated-policy-apply.adoc [hunk 1]
```diff
@@ -5,6 +5,20 @@ endif::[]
 
 Users with *Organization Admin*, *Environment Admin*, or with a role to *Manage Policies* in the environment can create, edit and delete Automated Policies.
 
+== Task Prerequisites
+
+Before you apply an automated policy, ensure that:
+
+* Mule runtime engine (Mule) is installed with xref:api-gateway-capabilities-mule4.adoc[API gateway capabilities] enabled.
+* An xref:api-instance-landing-page.adoc[API], configured with a basic endpoint or a proxy endpoint, exists in API Manager.
```
Comment (dhirsch1) [inline modules/ROOT/pages/automated-policy-apply.adoc]: I don't think you need the commas in this sentence.

File: modules/ROOT/pages/automated-policy-apply.adoc [hunk 2]
```diff
@@ -5,6 +5,20 @@ endif::[]
...
+* A Mule application is deployed with a xref:connectors::http/http-listener-ref.adoc[HTTP(s) based flow], which is linked to the API through Autodiscovery, also known as tracked flow.
```
Comment (dhirsch1) [inline modules/ROOT/pages/automated-policy-apply.adoc]: Does "tracked flow" refer to the Autodiscovery process or the deployment as a whole? This is not clear. If it refers to the deployment as a whole, I suggest something like this: "This deployment process is also called a tracked flow."

File: modules/ROOT/pages/offline-policy-task.adoc [hunk 3]
```diff
@@ -3,10 +3,23 @@ ifndef::env-site,env-github[]
...
+Mule detects the presence of the file and starts applying the policy. Mule generates JSON and XML policy files in a new directory in `%MULE_HOME%/policies`.
 
 *Note:* The information in this topic applies only to use with Mule 4.
 
+== Task Prerequisites
+
+Before you apply a default policy, ensure that:
+
+* Mule runtime engine (Mule) is installed with xref:api-gateway-capabilities-mule4.adoc[API gateway capabilities] enabled.
+* An xref:api-instance-landing-page.adoc[API], configured with a basic endpoint or a proxy endpoint, exists in API Manager.
```
Comment (dhirsch1) [inline modules/ROOT/pages/offline-policy-task.adoc]: Same comment about removing the commas.

File: modules/ROOT/pages/offline-policy-task.adoc [hunk 4]
```diff
@@ -3,10 +3,23 @@ ifndef::env-site,env-github[]
...
+* A Mule application is deployed with a xref:connectors::http/http-listener-ref.adoc[HTTP(s) based flow], which is linked to the API through Autodiscovery, also known as tracked flow.
```
Comment (dhirsch1) [inline modules/ROOT/pages/offline-policy-task.adoc]: Same comment as before.

File: modules/ROOT/pages/policies-policy-overview.adoc [hunk 5]
```diff
@@ -6,6 +6,16 @@ endif::[]
 
 Policies differ based on several different factors, such as category, purpose, version, and configuration options. Carefully consider these factors before you implement them in your environment. 
 
+Based on the Mule runtime engine (Mule), a policy Exchange asset is consists of:
+
+* Mule 4:
+** A deployable JAR file that contains the policy business logic.
+** A YAML configuration file, where the policy parameters and metadata are defined.
```
Comment (dhirsch1) [inline modules/ROOT/pages/policies-policy-overview.adoc]: where > in which

File: modules/ROOT/pages/using-policies.adoc [hunk 6]
```diff
@@ -4,24 +4,32 @@ include::_attributes.adoc[]
...
-* An API deployed to API Gateway Runtime 2.x
+By default, a policy applies to the entire API. 
+
+== Task Prerequisites
+
+Before you apply a default policy, ensure that:
 
-API Manager provides a number of policies. You can also build custom policies. You deploy a proxy API application and apply one or more policies to control how and when a received request is forwarded to its implementation endpoint. You can set an API alert to notify you when an API request violates a policy for SLA.
+* Mule runtime engine (Mule) is installed with xref:api-gateway-capabilities-mule4.adoc[API gateway capabilities] enabled.
+* An xref:api-instance-landing-page.adoc[API], configured with a basic endpoint or a proxy endpoint, exists in API Manager.
```
Comment (dhirsch1) [inline modules/ROOT/pages/using-policies.adoc]: Ditto

File: modules/ROOT/pages/using-policies.adoc [hunk 7]
```diff
@@ -4,24 +4,32 @@ include::_attributes.adoc[]
...
+* A Mule application is deployed with a xref:connectors::http/http-listener-ref.adoc[HTTP(s) based flow], which is linked to the API through Autodiscovery, also known as tracked flow.
```
Comment (dhirsch1) [inline modules/ROOT/pages/using-policies.adoc]: Ditto

File: modules/ROOT/pages/using-policies.adoc [hunk 8]
```diff
@@ -4,24 +4,32 @@ include::_attributes.adoc[]
...
+To apply a policy:
+
+* Go to *API Manager > API Administration* and select the API instance from the details window.
+* From the left navigation, select *Policies > Apply New Policy*.
++
+image::applying-default-policies.png[75%,75%]
+* Expand the policy that you want to apply.
+* Select the policy version and click *Configure Policy*.
++
+For details about the configuration parameters for each of the policy, see the specific xref:policies-ootb-landing-page.adoc[policy documentation].
```
Comment (dhirsch1) [inline modules/ROOT/pages/using-policies.adoc]: each of the policies

File: modules/ROOT/pages/automated-policy-apply.adoc (hunk 2 above)
Comment (nahuel-dallavecchia) [reply modules/ROOT/pages/automated-policy-apply.adoc]: A flow is said to be "tracked" if it has an associated autodiscovery element linking it to a valid API.

"When [autodiscovery](https://docs.mulesoft.com/api-manager/2.x/api-auto-discovery-new-concept) is correctly configured in your Mule application you can say that your application’s API is tracked by or paired to API Manager."

File: modules/ROOT/pages/offline-policy-task.adoc [hunk 9]
```diff
@@ -3,10 +3,23 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-An offline policy is applied directly to the runtime, not applied through API Manager. This procedure describes how to apply an offline policy to an API. You create a file in the offline-policies directory. Mule detects the presence of the file and starts applying the policy. Mule generates JSON and XML policy files in a new directory in `%MULE_HOME%/policies`.
+An offline policy is applied directly to the runtime and not through API Manager. This procedure describes how to apply an offline policy to an API. You create a file in the offline-policies directory.
```
Comment (dhirsch1) [inline modules/ROOT/pages/offline-policy-task.adoc]: The procedure should have steps:

1. Create a file in the offline-policies directory. 
2. Mule detects the presence of the file and starts applying the policy.
3. Mule generates JSON and XML policy files in a new directory in `%MULE_HOME%/policies`

Or, if you don't call it a procedure:

An offline policy is applied directly to the runtime and not through API Manager. To apply an offline policy to an API, create a file in the offline-policies directory. 

Mule detects the presence of the file and starts applying the policy. Mule then generates JSON and XML policy files in a new directory in `%MULE_HOME%/policies`.

File: modules/ROOT/pages/automated-policy-apply.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/automated-policy-apply.adoc]: Done! Thanks!

File: modules/ROOT/pages/automated-policy-apply.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/automated-policy-apply.adoc]: Done! Thanks!

File: modules/ROOT/pages/offline-policy-task.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/offline-policy-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/offline-policy-task.adoc (hunk 4 above)
Comment (priyaayyar) [reply modules/ROOT/pages/offline-policy-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/policies-policy-overview.adoc (hunk 5 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policies-policy-overview.adoc]: Done! Thanks!

File: modules/ROOT/pages/using-policies.adoc (hunk 6 above)
Comment (priyaayyar) [reply modules/ROOT/pages/using-policies.adoc]: Done! Thanks!

File: modules/ROOT/pages/using-policies.adoc (hunk 7 above)
Comment (priyaayyar) [reply modules/ROOT/pages/using-policies.adoc]: Done! Thanks!

File: modules/ROOT/pages/using-policies.adoc (hunk 8 above)
Comment (priyaayyar) [reply modules/ROOT/pages/using-policies.adoc]: good catch! Thanks! Done!

File: modules/ROOT/pages/offline-policy-task.adoc (hunk 9 above)
Comment (priyaayyar) [reply modules/ROOT/pages/offline-policy-task.adoc]: Did not add in steps but fixed as I deemed fit. This topic will go for a Kim edit later.

File: modules/ROOT/pages/offline-policy-task.adoc [hunk 10]
```diff
@@ -3,22 +3,38 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-An offline policy is applied directly to the runtime, not applied through API Manager. This procedure describes how to apply an offline policy to an API. You create a file in the offline-policies directory. Mule detects the presence of the file and starts applying the policy. Mule generates JSON and XML policy files in a new directory in `%MULE_HOME%/policies`.
+An offline policy is applied directly to the runtime and not through API Manager. This procedure describes how to apply an offline policy to an API. When you create a file in the offline-policies directory, Mule runtime engine (Mule) detects the presence of the file and starts applying the policy.
```
Comment (dhirsch1) [inline modules/ROOT/pages/offline-policy-task.adoc]: I would remove the sentence that starts with "This procedure...".

**Review by dhirsch1 — APPROVED** [review]
Approved with one suggestion.

File: modules/ROOT/pages/offline-policy-task.adoc (hunk 10 above)
Comment (priyaayyar) [reply modules/ROOT/pages/offline-policy-task.adoc]: Done! Thanks!

**Pull request merged**